js/components/addEditBookmark.js

```diff
@@ -16,9 +16,13 @@ class AddEditBookmark extends ImmutableComponent {
   onClose () {
     windowActions.setBookmarkDetail()
   }
+  componentDidMount () {
```

Comment:
this and the change in `addEditBookmarkHanger.js` are changes I missed when originally doing that control, which was submitted with https://github.com/brave/browser-laptop/pull/5118